tests/qir/test_example_qir_phi_int.ll

```diff
@@ -14,39 +14,47 @@ entry:
   call void @__quantum__qis__cnot__body(%Qubit* null, %Qubit* inttoptr (i64 1 to %Qubit*))
   call void @__quantum__qis__mz__body(%Qubit* null, %Result* null)
   %0 = call i1 @__quantum__qis__read_result__body(%Result* null)
-  %1 = zext i1 %0 to i2 
+  %1 = zext i1 %0 to i2    ; a[0] = %0
   call void @__quantum__qis__mz__body(%Qubit* inttoptr (i64 1 to %Qubit*), %Result* inttoptr (i64 1 to %Result*))
   %2 = call i1 @__quantum__qis__read_result__body(%Result* inttoptr (i64 1 to %Result*))
   %3 = zext i1 %2 to i2
   %4 = shl i2 %3, 1
-  %5 = or i2 %1, %4 
+  %5 = or i2 %1, %4 ; a[1] = %2
   %6 = trunc i2 %5 to i1
 
-  br i1 %6, label %then, label %continue
+  ; a = %5
+  ; b = 0
+  ; c = 125
 
-then:                                             ; preds = %entry
-  br label %continue
-
-continue:                                         ; preds = %entry, %then
   %7 = and i2 2, %5
   %8 = lshr i2 %7, 1
   %9 = trunc i2 %8 to i1
-  br i1 %9, label %then1, label %continue3
+  br i1 %9, label %then, label %continue; if a[2]
```

Comment:
a has only size 2, so a[2] is not defined?

Reply:
yes this an off by one, will fix.